src/Altinn.Apps/AppFrontend/react/altinn-app-frontend/src/components/base/ButtonComponent.tsx

```diff
@@ -11,7 +13,8 @@ export interface IButtonProvidedProps {
 }
 
 export function ButtonComponent(props: IButtonProvidedProps) {
-
+  const GetAutoSave = makeGetAutoSave();
```

Comment:
Do we need to use a pre-defined selector here when we are just getting the value directly from the state?

Reply:
I guess not, removing

Reply:
Fixed.